File: Java/12-Matrizes/Matriz2.java
class Matriz2 {
    public static void main(String args[]) {
        //Tambem podemos declarar e ja preencher a matriz
        int[][] matriz = {{1, 4},{2, 1}};

        //Para exibir a matriz em forma de tabela precisamos de duas estruturas de repetição
        for(int i = 0; i < matriz.length; i++) { //Varre linhas
            for(int j = 0; j < matriz[i].length; j++) { //Varre colunas
                System.out.print(matriz[i][j] +" ");
            }
            System.out.println();
        }
    }
}

File: PHP/12-Matrizes/Matriz2.php
<?php

    $matriz = array(array(6,4,2),
                    array(4,9,3),
                    array(3,2,4));
    //Temos tres vetores de 3 elementos cada um dentro de um vetor(matriz 3x3)

    //Para mostrar uma matriz em forma de tabela temos que aninhar
    //duas estruturas de repetição, uma para as linhas e outra
    //para as colunas

    foreach ($matriz as $subMatriz) {
        foreach($subMatriz as $elemento) {

            echo "$elemento ";

        }
        echo "<br/>";
    }

?>